src/lib/yang/snabb-softwire-v2.yang

```diff
@@ -723,6 +723,12 @@ module snabb-softwire-v2 {
          "Maximum packet size to recieve on the IPv6 interface.";
       }
 
+      leaf flow-label {
+        type uint32;
```

Comment:
This defaults to `0` right? Is it worth putting in an explicit `default: 0;` clause here or is that unnecesary? (I noticed some other leaves have such defaults)